### DISCUSSION TITLE
Hidden Files in e2 (idrive)

### DISCUSSION THREAD

**Discussion by ttammann** [iterate-ch/cyberduck#16726] [Q&A] [answered]
Hi

I use e2 from Idrive with versioning. When I rename a file, I can see the old name in e2 but dont know how to show these hidden files in mountain duck. In the screenshot, I see the AIFF files in mountain duck but can't see any .flac files anywhere...

thanks
Tom
<img width="1133" alt="Screenshot 2024-12-27 at 12 01 37" src="https://github.com/user-attachments/assets/85cdc805-9149-4b69-80d2-4cefe1d9093d" />

**Comment by dkocher** [comment, marked as answer]
Choose [_Info → Versions_](https://docs.mountainduck.io/mountainduck/info/#versions) to display a list of previous file versions in a S3 bucket.
